src/main/java/frc/robot/RobotContainer.java

```diff
@@ -398,21 +402,30 @@ public void setAutoPlacementLEDs(Optional<Alliance> alliance) {
     }
 
     // Light up in Shang Chi color if both translation and rotation are correct
-    if (rotationCorrect && XCorrect && YCorrect) {
+    if ((rotationCorrect && XCorrect && YCorrect) && (previousRotationCorrect != desiredRotationCorrect)) {
```

Comment:
More checks! More checks! (My way is probably not the best way to do it) but we want to also check the previousXCorrect and previousYCorrect to see if they match the current corresponding variables.